src/gleam_stdlib.erl

```diff
@@ -395,3 +396,14 @@ inspect_list([First | ImproperTail]) ->
 
 float_to_string(Float) when is_float(Float) ->
     erlang:iolist_to_binary(io_lib_format:fwrite_g(Float)).
+
+% Used to test side effects
+storage_init() ->
+    ets:new(gleam_test_storage, [ordered_set, protected, named_table]).
```

Comment:
This will fail if used a second time in the test suite because the name is already taken. Probably better to use the process dictionary

https://www.erlang.org/doc/man/erlang.html#put-2